src/submit-modal/success-content.js

```diff
@@ -81,14 +83,67 @@ const copyTableToClipboard = ({ importSummaries, requests }) => {
     navigator.clipboard.writeText(clipboardText)
 }
 
-const SummaryTable = ({ importSummaries }) => {
+const SummaryRow = ({
+    importSummary,
+    exchangeName,
+    expandedRows,
+    expandToggle,
+    hasConflicts,
+}) => {
+    return importSummary.conflicts?.length > 0 ? (
+        <DataTableRow
+            expanded={expandedRows.includes(exchangeName)}
+            onExpandToggle={() => expandToggle(exchangeName)}
+            expandableContent={
+                <ConflictsDetailsTable conflicts={importSummary.conflicts} />
+            }
+        >
+            <DataTableCell>{exchangeName}</DataTableCell>
+            <DataTableCell>
+                {importSummary?.importCount?.imported}
+            </DataTableCell>
+            <DataTableCell>{importSummary?.importCount?.updated}</DataTableCell>
+            <DataTableCell>{importSummary?.importCount?.ignored}</DataTableCell>
```

Comment:
A few too many `?` for my liking here. Even without knowing anything about the object shape, I can see that the `importSummary?.` is obsolete because on line 93 you're already doing `importSummary.`. Please remove at least the first `?` and reflect on whether or not the `.importCount?.` is needed.